kitsune/messages/jinja2/messages/outbox.html

```diff
@@ -14,81 +14,60 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
         </div>
       {% if msgs %}
         <form action="{{ url('messages.outbox_bulk_action') }}" method="POST">
-          <div class="sumo-button-wrap">
-            <button type="submit" name="delete" class="sumo-button">{{ _('Delete Selected') }}</button>
-          </div>
-          <ol class="message-list">
-              <li class="message-list--item">
-                <div class="field checkbox no-label">
-                </div>
-                <div class="avatar-details">
-                  <span class="to user">
-                    <b>{{ _('Sent') }}</b>
-                  </span>
-                  <span class="to user">
-                    <b>{{ _('To') }}</b>
-                  </span>                  
-                  {% if in_staff_group(request.user) %}
-                    <span class="to group">
-                        <b>{{ _('To Groups') }}</b>
-                    </span>
-                  {% endif %}
-                <span class="message-view">
-                    <b>{{ _('Message') }}</b>
-                </span>
-                <span class="delete-button">
-                </span>
-                </div>
-              </li>
+        <div class="outbox-table">
+            <div class="email-row header">
+                <div class="email-cell sent">{{ _('Sent On') }}</div>
+                <div class="email-cell to">{{ _('To Users') }}</div>
+                {% if in_staff_group(request.user) %}
+                    <div class="email-cell to-groups">{{ _('To Groups') }}</div>
+                {% endif %}
+                <div class="email-cell excerpt">{{ _('Excerpt') }}</div>
+                <div class="email-cell delete">{{ _('Delete?') }}</div>
+            </div>
+            <!-- Repeat the following div for each email message -->
             {% for message in msgs.object_list %}
-              <li class="message-list--item">
-                <div class="field checkbox no-label">
-                  <input type="checkbox" name="id" value="{{ message.id }}" id="id_checkbox_{{ message.id }}">
-                  <label for="id_checkbox_{{ message.id }}"></label>
-                </div>
-                <div class="avatar-details">
-                  <span class="time">
-                    <p>{{ datetimeformat(message.created) }}</p>
-                  </span>
-                  <span class="to user">
-
-                  {% for user in message.to.all()[:1] -%}
+            <div class="email-row">
+                <div class="email-cell sent">{{ datetimeformat(message.created) }}</div>
+                <div class="email-cell to">
+                {% for user in message.to.all()[:1] -%}
                     <a rel="nofollow" href="{{ profile_url(user) }}">
                       {%- if name -%}
                         {{ name }}
```

Comment:
As far as I can tell, `name` is never defined, so this chunk of code will never be executed.